src/targets/gpu/analyze_streams.cpp

```diff
@@ -38,17 +38,17 @@ struct hip_stream_model
     std::unordered_map<migraphx::instruction_ref, std::size_t> ins2stream{};
     std::size_t get_nstream() const { return max_stream + 1; }
     std::size_t get_stream(migraphx::instruction_ref ins) const { return ins2stream.at(ins); }
-    std::size_t get_event_id(migraphx::instruction_ref ins) const
+    static std::size_t get_event_id(migraphx::instruction_ref ins)
```

Comment:
This should not be static, so it matches the call in the interface.

Reply:
It matches the calling interface. Tool for static analysis suggested turning it to `static` or `constexpr` for better optimization.

Reply:
BTW, its a member function in the interface, see here: https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/src/include/migraphx/stream_model.hpp#L72.